scripts/haas_network.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=HaaS Network Server 
+After=network.target
+
+[Service]
+User=haas_user
+Group=haas_user
+WorkingDirectory=/var/lib/haas/
+ExecStart=/usr/bin/haas serve_networks
```

Comment:
So I will remove ``ExecReload`` directive. 
What should I do for ``KillMode=process`` ? 
Let me know if you have a better suggestion. 
Or else I will just add an comment explaining what it is doing.